src/main/java/team/creative/creativecore/common/util/math/box/BoxUtils.java

```diff
@@ -64,7 +64,7 @@ private static void includeMaxRotationInBox(ABB box, Vec3d vec, Axis axis, doubl
         
         if (rotation >= 90) {
             while (quarterRotation <= Math.abs(rotation) && quarterRotation < 360) {
-                Facing facing = positive ? state.clockwiseMaxFacing() : state.counterMaxClockwiseFacing();
+                Facing facing = state.clockwiseMaxFacing();
```

Comment:
Pretty sure this does not produce the same results.

Reply:
positive check is rotation > 0 and second check do the same but for 90, which is positive, and makes redundant check again if rotation is again positive

Reply:
yep, in the ``if check context`` we already do if rotation is positive, so there is no reason to check it again

![image](https://github.com/CreativeMD/CreativeCore/assets/29090346/9b7c1460-203d-4c79-90ed-78064d35c3b8)

Reply:
Rotation can be negative. In that case Positive is false.

Reply:
ok i read the change again and now i get your point

Reply:
i don't get it.

Reply:
Just revert to old.

Reply:
Inside the context of `rotation => 90` it can't be negative, never. Because rotation is over positive 90

The `positive ? :` check was done inside rotation => which already checks if rotation is positive

Also just mentioned, rotation is never modified, quarterRotation is

Reply:
I will revert it back, but is a redundand check in that context

Reply:
done